Makefile

```
@@ -165,22 +165,22 @@ ifeq ($(TAG),devel)
 	@$(BUILDER) pull ubuntu:20.04
 endif
 
-images = $(shell basename -s .Dockerfile -a build/docker/*.Dockerfile)
+images = $(shell basename -s .Dockerfile -a build/plugin-images/*.Dockerfile)
 
 skipbaselayercheck = intel-vpu-plugin intel-qat-plugin-kerneldrv intel-idxd-config-initcontainer
 distroless_images = $(patsubst %,$(REG)%\:$(TAG),$(filter-out $(skipbaselayercheck),$(images)))
 test-image-base-layer:
 	@for img in $(distroless_images); do scripts/test-image-base-layer.sh $$img $(BUILDER) || exit 1; done
 
 $(images):
-	@build/docker/build-image.sh $(REG)$@ $(BUILDER) $(EXTRA_BUILD_ARGS)
+	@build/build-image.sh $(REG)build/plugin-images/$@.Dockerfile $(BUILDER) $(EXTRA_BUILD_ARGS)
```

things could be simpler if 
```suggestion
	@cd build/plugin-images && ../build-image.sh $(REG)$@ $(BUILDER) $(EXTRA_BUILD_ARGS)
```

and `DOCKERFILE` becomes just `$(basename ${IMG}.Dockerfile)`. context does not need to change either with this approach. what do you think?

---

@mythi , thanks for the comment. I tried that, but it doesn't work with the following error, so it looks to me the plugin images require copying the entire repo to the docker build context and building from there, so we need to invoke the `docker build` from root of the repo:
```
...
Step 7/19 : WORKDIR $DIR
 ---> Running in 00702c766904
Removing intermediate container 00702c766904
 ---> d9c722c97482
Step 8/19 : COPY . .
 ---> 09b64f78c86e
Step 9/19 : RUN cd cmd/operator; GO111MODULE=${GO111MODULE} CGO_ENABLED=0 go install "${BUILDFLAGS}"; cd -
 ---> Running in fedde959ed1c
/bin/sh: 1: cd: can't cd to cmd/operator
go: go.mod file not found in current directory or any parent directory; see 'go help modules'
/intel-device-plugins-for-kubernetes
Removing intermediate container fedde959ed1c
```

---

ah, right. the build context must be set to the repo root: `$(dirname $0)/../`